plugins/scaffolder-backend/src/scaffolder/stages/types.ts

```diff
@@ -18,4 +18,5 @@ export type RemoteProtocol =
   | 'github'
   | 'gitlab'
   | 'gitlab/api'
-  | 'azure/api';
+  | 'azure/api'
+  | 'bitbucket/api';
```

Comment:
We want to ideally move away from thie `/api` pattern. So just use `bitbucket`

Reply:
Updated